<ui version="4.0" >
 <class>SettingsClass</class>
 <widget class="QDialog" name="SettingsClass" >
  <property name="windowModality" >
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>294</width>
    <height>349</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>294</width>
    <height>309</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>294</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Settings</string>
  </property>
  <widget class="QWidget" name="" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>271</width>
     <height>331</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2" >
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" >
      <item>
       <widget class="QGroupBox" name="groupBox" >
        <property name="title" >
         <string>Network</string>
        </property>
        <layout class="QGridLayout" name="gridLayout" >
         <item row="0" column="0" colspan="2" >
          <widget class="QRadioButton" name="radioButton" >
           <property name="text" >
            <string>Server</string>
           </property>
           <property name="checked" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2" >
          <widget class="QRadioButton" name="radioButton_2" >
           <property name="text" >
            <string>Client</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0" >
          <widget class="QLabel" name="label" >
           <property name="text" >
            <string>Address</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1" >
          <widget class="QLineEdit" name="lineEdit" >
           <property name="text" >
            <string>127.0.0.1</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0" >
          <widget class="QLabel" name="label_2" >
           <property name="text" >
            <string>Port</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1" >
          <widget class="QSpinBox" name="spinBox" >
           <property name="minimum" >
            <number>1</number>
           </property>
           <property name="maximum" >
            <number>65536</number>
           </property>
           <property name="value" >
            <number>2000</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2" >
        <property name="title" >
         <string>Gameplay</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2" >
         <item row="0" column="0" >
          <widget class="QLabel" name="label_5" >
           <property name="text" >
            <string>Max hit points</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" >
          <widget class="QSpinBox" name="spinBox_3" >
           <property name="minimum" >
            <number>1</number>
           </property>
           <property name="maximum" >
            <number>1000</number>
           </property>
           <property name="value" >
            <number>100</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0" >
          <widget class="QLabel" name="label_6" >
           <property name="text" >
            <string>Damage</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" >
          <widget class="QSpinBox" name="spinBox_4" >
           <property name="maximum" >
            <number>100</number>
           </property>
           <property name="value" >
            <number>20</number>
           </property>
          </widget>
         </item>
        </layout>
        <zorder>label_5</zorder>
        <zorder>lineEdit_3</zorder>
        <zorder>label_6</zorder>
        <zorder>spinBox_3</zorder>
        <zorder>spinBox_4</zorder>
        <zorder>pushButton</zorder>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="layoutDirection" >
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text" >
       <string>OK</string>
      </property>
      <property name="default" >
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
